src/build.d

```diff
@@ -209,7 +209,7 @@ alias lexer = memoize!(function()
             "-of" ~ target,
             "-lib",
             "-vtls",
-            "-J"~env["G"], "-J../res",
+            "-J" ~ env["RES"],
```

Comment:
As far as I can see there are only two files that are string-imported (and for which we need the `-J` flag is needed):
* `VERSION`, used by https://github.com/dlang/dmd/blob/a7017a9634a33d6234d28b60bc96a1a09e5d704c/src/dmd/globals.d#L355 and produced by the `versionFile` target and placed in `env["G"]`. `globals.d` is used by the `lexer` target, so I think we need to keep it.
* `res/default_ddoc_theme.ddoc`, used by https://github.com/dlang/dmd/blob/a7017a9634a33d6234d28b60bc96a1a09e5d704c/src/dmd/doc.d#L366
`doc.d` is part of the `frontend` source, but not of the `lexer` ones, so I think we should drop `env["RES"]` here.

Reply:
`globals.d` is not imported by the lexer. That's why the import can be removed here.

Reply:
But why is it part of the `lexer` sources then?

Reply:
Ok I've removed the res import path from `lexer` and it still appears to build.  Nice catch.